### ISSUE TITLE
Not an actual issue.

### ISSUE THREAD

**Issue by RawTechnique** [LWSS/Ape-ex-Abominations#12]
Hi, I am learning and I am wondering if you would have any time to talk about your QEMU DMA cheat.

**Comment by LWSS** [comment]
yeah ok what do u want to know?

**Comment by RawTechnique** [comment]
Is there any way would it be possible to directly write to video buffer or to an external program like looking glass without adding more detection vectors than reading from memory. I know parsec has a opengl implementation, but that has a lot of downsides.

**Comment by LWSS** [comment]
you mean like this? (https://github.com/LWSS/peeper)

Writing to the video buffer would be a huge pain and specific to your driver afaik.

**Comment by RawTechnique** [comment]
Thank you! That is exactly what I meant! and oh ok, I did not know that would be such a big task.

**Comment by LWSS** [comment]
cool bro, good luck

**Comment by RawTechnique** [comment]
Would you want to help work on a modified Qemu that randomizes (acpi tables, version data, device names, manufacturer information, model numbers, and drive names for each machine) I know its simple, but I cant find a single one available that does it at all or well and without manual input. I made a simple script that does most of those things and fixes all detected issues in al-khaser and Pafish as well as bypassing NVidias error 43, and NVidias mobile GPU battery detection/battery monitoring probe.

**Comment by LWSS** [comment]
I actually didn't know about al-khaser/pafish, thanks.

Well I have never ran into vm-detection issues from any of the anti-cheats, blizzard, easyAC, BE, etc... This could be an issue for valorant however, apparently they have 3 different vm detection methods or so if I remember the thread on uc right.

The only one I have looked into a little is cpuid, and from what I read, it could be bypassed by changing the configuration to spoof the cpuid. Also I know you can change the hdd names here too.

I think it would probably be ideal to do as much as you can without forking qemu because from my experience people don't like building big stuff, they would rather hook the installed program than rebuild it from source. (this is why I didn't fork looking-glass for peeper)

**Comment by RawTechnique** [comment]
I ran into the vm issue when trying doing a bug bounty on ESEA (I know I probably won't find anything but its fun to learn), and I'll have to see if Riot has a bug bounty for Valorant since that seems cool to look at if they have 3 detection methods, also you cant spoof the cpuid on Qemu due to their cpu clock emulation implementation.

**Comment by LWSS** [comment]
Yeah let me know if you come up with something, it sounds useful in the future. I want to make a tool like Reclass that goes into qemu memory. It sucks not having it lol.

**Comment by RawTechnique** [comment]
Also holy shit Riot seems like a fucking shitty company pardon my language (after reading UC and fact checking it) I am doing it as soon as I get a key screw the bug bounty.

**Comment by RawTechnique** [comment]
Also reclass in Qemu memory would be amazing

**Comment by LWSS** [comment]
yeah i'm friends with heep and have used kvm_explorer, but it kinda stinks, can't deref pointers and doesn't have adjustable view. He was going to make a reclass fork but i'm not sure if he finished it.

**Comment by RawTechnique** [comment]
Oh thats cool, I hope he releases it if is finished!

**Comment by RawTechnique** [comment]
Do you have a Discord?

**Comment by LWSS** [comment]
yeah sorry I got my fingers in a lotta pies and sometimes I dont take the time to lick em
`It's been one-a those days#4849`
